api/v1alpha1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1alpha1 contains API Schema definitions for the certification v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=certification.redhat.com
```

Comment:
I don't think you want a space here. IIRC, markers are commonly distinguished from comments _because_ they lack this space. I'm not sure if it's enforced or a semantic, though.

Reply:
Another common example is
```
//go:embed ...
```

Reply:
fixed, something was auto formatting this, still don't know what.